Documentation/microsoft-team.md

```diff
@@ -0,0 +1,23 @@
+# Microsoft Team Onboarding
+
+If you are new to Microsoft, you will need to onboard into various GitHub projects in order to get your work done.
+
+## Getting Permissions
+
+There are two main teams to join to get access to Microsoft .NET-related projects.
+
+* [microsoft team - dotnet org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
+* [dotnet team - microsoft org](https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/)
+
```

Comment:
Can we add ask to:
1. Make membership public information (they will show as badges on your profile).
2. Make sure either GH name matches GAL name, or make your MS email public -- motivation: other MS employees can recognize you & find you internally if they need to follow up.
3. Add picture (showing face to community for better communication).

Reply:
Add picture for sure.

Reply:
Just to make sure: for 2. by GH name you mean the full name, not the alias right?